### PR TITLE
Visibility setting to gone and setting up the recyclerView

### PR DESCRIPTION
**Pull request by adrishyantee**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu Jul 16 02:18:59 IST 2020
+#Sun Dec 06 19:05:41 IST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (s-ayush2903) [inline gradle/wrapper/gradle-wrapper.properties:6]: Revert the changes in this file

File: build.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:4.0.0"
+    classpath 'com.android.tools.build:gradle:4.1.1'
```
Comment (s-ayush2903) [inline build.gradle:11]: We don't need this upgrade, revert it

**Comment by s-ayush2903** [comment]
Also, you need to create an adapter to bind the data to the recyclerView, then it'll be displayed on the screen. Also, you'll have to add some placeholder data maybe textViews and(or) an ImageView for that to be displayed on the screen.
You may read [this](https://developer.android.com/courses/pathways/android-basics-kotlin-unit-2-pathway-2) if need some assistance.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (adrishyantee) [reply gradle/wrapper/gradle-wrapper.properties:6]: Minimum supported Gradle version is 6.5. Current version is 6.1.1. (there's an error I received)

File: build.gradle (hunk 2 above)
Comment (adrishyantee) [reply build.gradle:11]: revert done

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (adrishyantee) [reply gradle/wrapper/gradle-wrapper.properties:6]: reverted
